webapp/src/components/preview-movies/style/poster/Poster.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useRef, useState } from 'react';
+import classNames from 'classnames';
+import YouTube from 'react-youtube';
+import { getGlobalInstance } from 'plume-ts-di';
+import Player from '../../../general/streaming/movie/Player';
+import PosterBackground from './PosterBackground';
+import TrailerService from '../../../../services/streaming/TrailerService';
+import { MediaType, Trailer } from '../../../../api/types/MovieDbTypes';
+import useLoader from '../../../../lib/plume-http-react-hook-loader/promiseLoaderHook';
+import MediaDetails from './MediaDetails';
+import ShowLargeTrailer from '../../../general/streaming/trailer/ShowLargeTrailer';
+
+type Props = {
+  title: string,
+  overview: string,
+  id: number,
+  type: string,
+  backdrop_path: string,
+  isSelected: boolean,
+  stopInterval: () => void
+};
+
+const SHOW_TRAILER_TIMER: number = 1000;
```

Comment:
pour plus de lisibilité :

```suggestion
const SHOW_TRAILER_TIMER: number = 1_000;
```